.sass-lint.yml

```diff
@@ -89,6 +87,7 @@ rules:
   space-before-bang: 1
   space-after-bang: 1
   space-between-parens: 1
+  space-around-operator: 0 # because it's hard to read when you have multiple value set with operators (eg. padding)
```

Comment:
I think it's more readable when set to 1. Any counter-example?

Reply:
Yes sure:
```sass
padding: 13 * $var 13 * $var; // this is what the rule would enforce
// vs
padding: 13*$var 14*$var;
```

Reply:
Oh ok, nevermind, thought `0` would enforce no space. I'm good with this then 👍